content/ar/docs/concepts/overview/_index.md

```diff
@@ -0,0 +1,96 @@
+---
+# reviewers:
+# - bgrant0607
+# - mikedanese
+title: "ملخص"
+description: >
+  كوبيرنيتيس هو نظام محمول قابل للتوسع ومفتوح المصدر، مخصص لإدارة التطبيقات والخدمات التي تعمل داخل الحاويات، حيث يسهل الإعداد التصريحي و التنفيذ الآلي. يتمتع كوبيرنيتيس بنظام بيئي متكامل واسع وسريع النمو. كما أن خدمات كوبيرنيتيس من دعم و أدوات المتوفرة على نطاق واسع.
+content_type: concept
+weight: 20
+card:
+  name: concepts
+  weight: 10
+no_list: true
+---
+
+<!-- overview -->
+هذه الصفحة هي نظرة عامة على كوبيرنيتيس.
+
+
+<!-- body -->
+  كوبيرنيتيس هو نظام محمول قابل للتوسع ومفتوح المصدر، مخصص لإدارة التطبيقات والخدمات التي تعمل داخل الحاويات، حيث يسهل الإعداد التصريحي و التنفيذ الآلي. يتمتع كوبيرنيتيس بنظام بيئي متكامل واسع وسريع النمو. كما أن خدمات كوبيرنيتيس من دعم و أدوات متوفرة على نطاق واسع.
+
+نشأ اسم كوبيرنيتيس من اليونانية، و يعني قائد دفة السفينة. ويختصر ك8س (تنطق كيتس) كناية عن الأحرف الثمانية بين الكاف والسين. فتحت جوجل مصدر مشروع كوبيرنيتيس في عام 2014. و هو حصيلة  [أكثر من 15 عامًا من خبرة جوجل](/blog/2015/04/borg-predecessor-to-kubernetes/) إلى جانب أفضل الأفكار والممارسات من المجتمع.
+
+## الرجوع بالزمن
+
+دعنا نلقي نظرة على فائدة كوبيرنيتيس من خلال الرجوع بالزمن.
+
+![Deployment evolution](/ar/docs/images/Container_Evolution.svg)
```

Comment:
Ideally, localize the `alt` attribute (“Deployment evolution”)

Reply:
nice catch done